constants.py

```diff
@@ -1,17 +1,19 @@
 import discord
 
+import config
+
 HELP_REPLY = discord.Embed(
     title="Ferris Elf help page",
     color=0xE84611,
-    description="""
+    description=f"""
 **help** - Send this message
 **info** - Some useful information about benchmarking
 **best _[day]_ _[part]_** - Best times so far for a day
 **submit _[day]_ _[part]_ <attachment>** - Benchmark attached code
 
 If [_day_] and/or [_part_] is ommited, they are assumed to be today and part 1
 
-Message <@{}> for any questions""",
+Message <@{config.settings.discord.owner_id}> for any questions""",
```

Comment:
A change like this, but much more thorough, is in https://github.com/proegssilb/ferris-elf/pull/53 . Can we undo the changes in this file to minimize merge conflicts?

Reply:
fair, can do